pkg/chartmuseum/server.go

```diff
@@ -27,20 +29,30 @@ type (
 	Router struct {
 		*gin.Engine
 	}
-
+	fetchedObjects struct {
+		objects []storage.Object
+		err     error
+	}
+	indexRegeneration struct {
+		index *repo.Index
+		err   error
+	}
```

Comment:
lets order stuff starting with lowercase either above or below uppercase, just not in the middle

Reply:
done.